src/app/templates/app/license_requests.html

```diff
@@ -70,14 +73,29 @@
     });
     $('body').on('click', 'button[id^="archive_button"]', function (en) {
       en.stopImmediatePropagation();
-      $("#modal-header").addClass("yellow-modal");
-	    $(".modal-footer").html('<button class="btn btn-success" id="confirm_archive"><span class="glyphicon glyphicon-off"></span> Confirm</button>');
-      $("#modal-body").html('Are you sure you want to archive this license?');
-      $("#myModal").modal({
-        backdrop: 'static',
-        keyboard: true,
-        show: true,
-      })
+      var githubLogin =  "{{ github_login }}" ;
+      if ((githubLogin) != "None"){
+        if (" {{ authorized }} ") {
```

Comment:
Why are there always two checks? if `githubLogin` and `authorized` ? can this be combined?

Reply:
basically github login check is for whether the person is logged in by github or not. A person not logged in by github cant be checked for permissions. Authorisation check is for whether the person is a collaborator or not.